### PR TITLE
Add start of list of formatted physics symbols for IMAS quantities

### PR DESCRIPTION
**Pull request by eldond**
- Goal: make it easy to get nicely formatted physics symbols for quantities

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,9 +62,12 @@ json:
 	cd omas/utilities && python3 build_json_structures.py
 	make cocos
 
-cocos:
+cocos: symbols
```
Comment (eldond) [inline Makefile:65]: @orso82 Is this right? Seems like the next entry might've bled through here by accident, but I'm not sure.

**Comment by eldond** [comment]
@orso82 Okay, great. This makes it pretty clear what needs to be done to extend the symbols later. Then to use the symbols, I just import PhysicsSymbols, right? OOHHH and you made it so I can just feed in whatever location I want without bothering to strip/split/whatever it myself, didn't you!? So convenient!

File: omas/tests/test_omas_core.py [hunk 2]
```diff
@@ -427,6 +427,20 @@ def test_codeparameters(self):
         assert len(ods['ec_launchers.code.parameters.launcher.0']) == 2
         return
 
+    def test_latexit(self):
+        assert latexit['somewhere.:.sublocation.n_e'] == '$n_e$'
+        assert latexit['n_e'] == '$n_e$'
+        assert latexit['.n_e'] == '$n_e$'
+        assert latexit['n_e'] == '$n_e$'
```
Comment (smithsp) [inline omas/tests/test_omas_core.py:434]: What is the difference between this and two lines prior?

**Comment by eldond** [comment]
@orso82 I'm finding it pretty convenient that you rigged it so I can do `latexit[thing]` instead of `latexit(thing)`, because I can also do `latexit.get(thing, thing)` and that's really clear.

**Pull request merged**